src/hooks/useList.js

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+import { getList } from '../utils/firebaseUtils';
+import { useLocalStorage } from './useLocalStorage';
+
+export const useList = () => {
```

Comment:
I'm honest. I'm a little confused with your custom hook `useList`.

1. What's is the difference between `useProducts` & `useList`? I guess both get values from Firebase.